### PR TITLE
feat: add-inv-function

### PR DESCRIPTION
**Pull request by andybeltz**
### Summary

Added `Update-AriaNetworksvCenterDataSourceCredentials` cmdlet to update the credentials for a vCenter Server in VMware Aria Operations for Networks.

### Type

- [ ] Bugfix
- [X] Enhancement or Feature
- [ ] Code Style or Formatting
- [ ] Documentation
- [ ] Refactoring
- [ ] Chore
- [ ] Other
        Please describe:

### Breaking Changes?

- [ ] Yes, there are breaking changes.
- [X] No, there are no breaking changes.

### Test and Documentation

- [X] Tests have been completed.
- [X] Documentation has been added or updated.

### CODE REVIEW

File: PowerValidatedSolutions.psm1 [hunk 1]
```diff
@@ -51698,6 +51698,70 @@ Function New-AriaNetworksNsxtDataSource {
...
+Function Update-AriaNetworksvCenterDataSourceCredentials {
+    <#
+        .SYNOPSIS
+        Update a configured vCenter Server data source's credentials in VMware Aria Operations for Networks.
+
+        .DESCRIPTION
+        The Update-AriaNetworksvCenterDataSourceCredentials cmdlet allows a user to update a previously configured vCenter Servers credentials in VMware Aria Operations for Networks.
+
+        .EXAMPLE
+        Update-AriaNetworksvCenterDataSourceCredentials -id 15832:902:2623605245375371420 -CollectorId 15832:901:1711011916294613031 -fqdn sfo-m01-vc01.sfo.rainpole.io -username svc-inv-vsphere -password VMw@re1!  -nickname "sfo-m01-vc01 - Management Domain vCenter Server"
```
Comment (GaryJBlake) [inline PowerValidatedSolutions.psm1:51711]:
```suggestion
        Update-AriaNetworksvCenterDataSourceCredentials -id 15832:902:2623605245375371420 -CollectorId 15832:901:1711011916294613031 -fqdn sfo-m01-vc01.sfo.rainpole.io -username svc-inv-vsphere -password VMw@re1! -nickname "sfo-m01-vc01 - Management Domain vCenter Server"
```

File: PowerValidatedSolutions.psm1 [hunk 2]
```diff
@@ -51698,6 +51698,70 @@ Function New-AriaNetworksNsxtDataSource {
...
+        This example updates a configured vCenter Server with updated values as supplied in VMware Aria Operations for Networks.
```
Comment (GaryJBlake) [inline PowerValidatedSolutions.psm1:51712]:
```suggestion
        This example updates a configured vCenter Server with updated credentials as supplied in VMware Aria Operations for Networks.
```

File: docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+## Synopsis
+
+Update a configured vCenter Server data source's credentials in VMware Aria Operations for Networks.
+
+## Syntax
+
+```powershell
+Update-AriaNetworksvCenterDataSourceCredentials [-id] <String> [-collectorId] <String> [[-fqdn] <String>]
+ [-nickname] <String> [-username] <String> [-password] <String> [<CommonParameters>]
```
Comment (GaryJBlake) [inline docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md:11]:
```suggestion
Update-AriaNetworksvCenterDataSourceCredentials [-id] <String> [-collectorId] <String> [[-fqdn] <String>] [-nickname] <String> [-username] <String> [-password] <String> [<CommonParameters>]
```

File: docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+## Description
+
+The `Update-AriaNetworksvCenterDataSourceCredentials` cmdlet allows a user to update a previously configured vCenter Servers credentials in VMware Aria Operations for Networks.
+
+## Examples
+
+### Example 1
+
+```powershell
+Update-AriaNetworksvCenterDataSourceCredentials -id 15832:902:2623605245375371420 -CollectorId 15832:901:1711011916294613031 -fqdn sfo-m01-vc01.sfo.rainpole.io -username svc-inv-vsphere -password VMw@re1!  -nickname "sfo-m01-vc01 - Management Domain vCenter Server"
```
Comment (GaryJBlake) [inline docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md:23]:
```suggestion
Update-AriaNetworksvCenterDataSourceCredentials -id 15832:902:2623605245375371420 -CollectorId 15832:901:1711011916294613031 -fqdn sfo-m01-vc01.sfo.rainpole.io -username svc-inv-vsphere -password VMw@re1! -nickname "sfo-m01-vc01 - Management Domain vCenter Server"

```

File: docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+```
+
+This example updates a configured vCenter Server with updated values as supplied in VMware Aria Operations for Networks.
```
Comment (GaryJBlake) [inline docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md:26]:
```suggestion
This example updates a configured vCenter Server with updated credentials as supplied in VMware Aria Operations for Networks.

```

**Review by GaryJBlake — CHANGES_REQUESTED** [review]
Couple of minor issues

File: PowerValidatedSolutions.psm1 [hunk 6]
```diff
@@ -51698,6 +51698,70 @@ Function New-AriaNetworksNsxtDataSource {
     }
 }
 Export-ModuleMember -Function New-AriaNetworksNsxtDataSource
+
+Function Update-AriaNetworksvCenterDataSourceCredentials {
+    <#
+        .SYNOPSIS
+        Update a configured vCenter Server data source's credentials in VMware Aria Operations for Networks.
```
Comment (tenthirtyam) [inline PowerValidatedSolutions.psm1:51705]:
```suggestion
        Update credentials for a vCenter Server data source credentials in VMware Aria Operations for Networks.
```

File: PowerValidatedSolutions.psm1 (hunk 2 above)
Comment (tenthirtyam) [reply PowerValidatedSolutions.psm1:51712]:
```suggestion
        This example updates the credentials for vCenter Server data source in VMware Aria Operations for Networks.
```

File: PowerValidatedSolutions.psm1 [hunk 7]
```diff
@@ -51698,6 +51698,70 @@ Function New-AriaNetworksNsxtDataSource {
...
+
+    Param (
+        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$id,
+        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$collectorId,
+		[Parameter (Mandatory = $false)] [ValidateNotNullOrEmpty()] [String]$fqdn,
+        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$nickname,
+		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$username,
+		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$password
+
+    )
```
Comment (tenthirtyam) [inline PowerValidatedSolutions.psm1:51741]:
```suggestion
		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$password
    )
```

File: docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md (hunk 5 above)
Comment (tenthirtyam) [reply docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md:26]:
```suggestion
This example updates the credentials for vCenter Server data source in VMware Aria Operations for Networks.

```

File: PowerValidatedSolutions.psm1 (hunk 1 above)
Comment (andybeltz) [reply PowerValidatedSolutions.psm1:51711]: Fixed in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/7344af3d25523db031599dbdd47c90a7d78ba0d9

File: PowerValidatedSolutions.psm1 (hunk 2 above)
Comment (andybeltz) [reply PowerValidatedSolutions.psm1:51712]: Fixed in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/7344af3d25523db031599dbdd47c90a7d78ba0d9

File: docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md (hunk 3 above)
Comment (andybeltz) [reply docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md]: Fixed in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/7344af3d25523db031599dbdd47c90a7d78ba0d9

File: docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md (hunk 4 above)
Comment (andybeltz) [reply docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md]: Fixed in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/7344af3d25523db031599dbdd47c90a7d78ba0d9

File: docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md (hunk 5 above)
Comment (andybeltz) [reply docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md:29]: Fixed in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/7344af3d25523db031599dbdd47c90a7d78ba0d9

File: PowerValidatedSolutions.psm1 [hunk 8]
```diff
@@ -51698,6 +51698,70 @@ Function New-AriaNetworksNsxtDataSource {
...
 }
 Export-ModuleMember -Function New-AriaNetworksNsxtDataSource
+
+Function Update-AriaNetworksvCenterDataSourceCredentials {
+    <#
+        .SYNOPSIS
+        Update a configured vCenter Server data source's credentials in VMware Aria Operations for Networks.
+
+        .DESCRIPTION
+        The Update-AriaNetworksvCenterDataSourceCredentials cmdlet allows a user to update a previously configured vCenter Servers credentials in VMware Aria Operations for Networks.
```
[the remark this answers is not in the document]
Comment (andybeltz) [reply PowerValidatedSolutions.psm1:51708]: Fixed in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/7344af3d25523db031599dbdd47c90a7d78ba0d9

File: docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md [hunk 9]
```diff
@@ -0,0 +1,128 @@
+# Update-AriaNetworksvCenterDataSourceCredentials
+
+## Synopsis
+
+Update a configured vCenter Server data source's credentials in VMware Aria Operations for Networks.
```
[the remark this answers is not in the document]
Comment (andybeltz) [reply docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md]: Fixed in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/7344af3d25523db031599dbdd47c90a7d78ba0d9

File: docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md [hunk 10]
```diff
@@ -0,0 +1,131 @@
...
+## Syntax
+
+Update-AriaNetworksvCenterDataSourceCredentials [-id] <String> [-collectorId] <String> [[-fqdn] <String>] [-nickname] <String> [-username] <String> [-password] <String> [<CommonParameters>]
+ [-nickname] <String> [-username] <String> [-password] <String> [<CommonParameters>]
+```
+
+## Description
+The `Update-AriaNetworksvCenterDataSourceCredentials` cmdlet allows a user to update credentials for a vCenter Server data source in VMware Aria Operations for Networks.
+
+The `Update-AriaNetworksvCenterDataSourceCredentials` cmdlet allows a user to update a previously configured vCenter Servers credentials in VMware Aria Operations for Networks.
```
Comment (tenthirtyam) [inline docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md:17]:
```suggestion
## Description

The `Update-AriaNetworksvCenterDataSourceCredentials` cmdlet allows a user to update credentials for a vCenter Server data source in VMware Aria Operations for Networks.

```

File: docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md [hunk 11]
```diff
@@ -0,0 +1,131 @@
...
+
+### Example 1
+
+Update-AriaNetworksvCenterDataSourceCredentials -id 15832:902:2623605245375371420 -CollectorId 15832:901:1711011916294613031 -fqdn sfo-m01-vc01.sfo.rainpole.io -username svc-inv-vsphere -password VMw@re1! -nickname "sfo-m01-vc01 - Management Domain vCenter Server"
+
+This example updates a configured vCenter Server with updated credentials as supplied in VMware Aria Operations for Networks.
+
+```
+
+This example updates a configured vCenter Server with updated values as supplied in VMware Aria Operations for Networks.
```
Comment (tenthirtyam) [inline docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md:29]:
```suggestion
### Example 1

Update-AriaNetworksvCenterDataSourceCredentials -id 15832:902:2623605245375371420 -CollectorId 15832:901:1711011916294613031 -fqdn sfo-m01-vc01.sfo.rainpole.io -username svc-inv-vsphere -password VMw@re1! -nickname "sfo-m01-vc01 - Management Domain vCenter Server"
```

This example updates the credentials for vCenter Server data source in VMware Aria Operations for Networks.

```

Comment (tenthirtyam) [inline docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md:29]:
```suggestion
### Example 1

Update-AriaNetworksvCenterDataSourceCredentials -id 15832:902:2623605245375371420 -CollectorId 15832:901:1711011916294613031 -fqdn sfo-m01-vc01.sfo.rainpole.io -username svc-inv-vsphere -password VMw@re1! -nickname "sfo-m01-vc01 - Management Domain vCenter Server"

This example updates a configured vCenter Server with updated credentials as supplied in VMware Aria Operations for Networks.

```

This example updates the credentials for vCenter Server data source in VMware Aria Operations for Networks.

```

File: docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md [hunk 12]
```diff
@@ -0,0 +1,131 @@
...
+
+The `Update-AriaNetworksvCenterDataSourceCredentials` cmdlet allows a user to update a previously configured vCenter Servers credentials in VMware Aria Operations for Networks.
+
+## Examples
+
+### Example 1
+
+Update-AriaNetworksvCenterDataSourceCredentials -id 15832:902:2623605245375371420 -CollectorId 15832:901:1711011916294613031 -fqdn sfo-m01-vc01.sfo.rainpole.io -username svc-inv-vsphere -password VMw@re1! -nickname "sfo-m01-vc01 - Management Domain vCenter Server"
+
+This example updates a configured vCenter Server with updated credentials as supplied in VMware Aria Operations for Networks.
```
Comment (tenthirtyam) [inline docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md:25]:
```suggestion
Update-AriaNetworksvCenterDataSourceCredentials -id 15832:902:2623605245375371420 -CollectorId 15832:901:1711011916294613031 -fqdn sfo-m01-vc01.sfo.rainpole.io -username svc-inv-vsphere -password VMw@re1! -nickname "sfo-m01-vc01 - Management Domain vCenter Server"

```

**Review by tenthirtyam — CHANGES_REQUESTED** [review]
Minor updates still needed, but almost done.

File: docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md [hunk 13]
```diff
@@ -0,0 +1,131 @@
+# Update-AriaNetworksvCenterDataSourceCredentials
+
+## Synopsis
+
+Update credentials for a vCenter Server data source credentials in VMware Aria Operations for Networks.
+
+
+## Syntax
```
[the remark this answers is not in the document]
Comment (andybeltz) [reply docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md]: Fixed in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/7229fe225c563ec7552b29fae42e0030b7454640

File: docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md (hunk 10 above)
Comment (andybeltz) [reply docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md]: Fixed in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/7229fe225c563ec7552b29fae42e0030b7454640

File: docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md (hunk 11 above)
Comment (andybeltz) [reply docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md]: Fixed in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/7229fe225c563ec7552b29fae42e0030b7454640

File: docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md (hunk 12 above)
Comment (andybeltz) [reply docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md:25]: Fixed in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/7229fe225c563ec7552b29fae42e0030b7454640

**Review by tenthirtyam — APPROVED** [review]
LGTM! :shipit:

**Pull request merged**